eos/capSim.py

```diff
@@ -135,13 +132,13 @@ def run(self):
         capCapacity = self.capacitorCapacity
         tau = self.capacitorRecharge / 5.0
 
-        cap_wrap = capCapacity                # cap value at last period
-        cap_lowest = capCapacity            # lowest cap value encountered
-        cap_lowest_pre = capCapacity    # lowest cap value before activations
-        cap = capCapacity                         # current cap value
-        t_wrap = self.period                    # point in time of next period
+        cap_wrap = capCapacity  # cap value at last period
+        cap_lowest = capCapacity  # lowest cap value encountered
+        cap_lowest_pre = capCapacity  # lowest cap value before activations
+        cap = capCapacity  # current cap value
+        t_wrap = self.period  # point in time of next period
 
-        t_now = t_last = 0
```

Comment:
So we had:

```
        t_now = t_last = 0
        t_max = self.t_max

        while 1:
            activation = pop(state)
            t_now, duration, capNeed, shot, clipSize = activation
            if t_now >= t_max:
                break
```

And it got changed to:

```
        t_last = 0
        t_max = self.t_max

        while 1:
            activation = pop(state)
            t_now, duration, capNeed, shot, clipSize = activation
            if t_now >= t_max:
                break
```

t_now is assigned when we take activation and break it down.  The creation/assignment of it when we assign `t_last` wasn't doing anything, pyCharm complained that it's a variable that's assigned but never used, and I did some quick validation.  Basically we're assigning a 0 to the var, then just blowing that assignment away when we do `t_now, duration, capNeed, shot, clipSize = activation`.

Doesn't _hurt_ to have, but not necessary.  We don't do assignments for `duration`, `capNeed`, `shot`, or `clipSize` either.